### PR TITLE
Feature/ntrnl 63 improve node scripts to better check for vulnerability

### PR DESCRIPTION
**Pull request by Mouhajer-CO**
### JIRA link

Resolves [NTRNL-63](https://technologyprogramme.atlassian.net/browse/NTRNL-63) and [NTRNL-12](https://technologyprogramme.atlassian.net/browse/NTRNL-12)

### Change description

- Improve node scripts to better check for vulnerability
- Identify All Github Users & Teams
- Add clone and state scripts
- Improve infrastructure scripts and state files

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -19,6 +19,8 @@ DEP_CHECK_OUT_FILE_NAME := $(PREFIX)__$(ORG_NAME)__$(REPO_NAME)__$(BRANCH_NAME)_
 
 DOCKER_COMPOSE_OUT_FILE_NAME := ./infrastructure/output/$(PREFIX)__docker__compose__output__$(TIMESTAMP).txt
 
+NOVE_VERSION := v18.16.0
```
Comment (Harrisman05) [inline Makefile:22]: Minor typo, this should be NODE_VERSION.

File: Makefile [hunk 2]
```diff
@@ -34,6 +36,7 @@ dependency-checks:
 	npm audit --json > $(DEP_CHECK_OUT_FILE_NAME)
 
 start:
+	$(info Node version: $(NOVE_VERSION))
```
Comment (Harrisman05) [inline Makefile:39]: Same typo

File: Makefile (hunk 1 above)
Comment (Mouhajer-CO) [reply Makefile:22]: Thanks @Harrisman05 will do 👍

**Review by Harrisman05 — APPROVED** [review]
Looks good to me. Typo amended and paired while running through scripts locally.

**Pull request merged**